global.json

```diff
@@ -1,5 +1,9 @@
 {
   "msbuild-sdks": {
-    "MSBuild.Sdk.Extras": "2.1.2"
+    "Uno.Sdk": "6.4.0-dev.50",
+    "Uno.Sdk.Private": "6.4.0-dev.256"
```

Comment:
Usually we don't use both in a same solution or it would cause strange problems with the DevServer.

I suggestion keeping only `Uno.Sdk.Private` here.

Reply:
Note: [`Uno.Sdk 6.4.0-dev.50` uses Uno version `6.4.0-dev.235`](https://www.nuget.org/packages/Uno.Sdk/6.4.0-dev.50#readme-body-tab), not `6.4.0-dev.256`